draft-demarco-status-attestations.md

```diff
@@ -186,8 +189,8 @@ Verifier, and not expired.
 
 # Requirements
 
-In this section are listed the general requirements that must be satisfied
-when the Status Attestation are implemented. The Status Attestation:
+The general requirements that MUST be met while implementing the Status Attestation are listed in this section.  
```

Comment:
the MUST is listed in each requirement, there may be some requirement in the future with SHOULD. That's why I prefer to use the normative language within each point

Reply:
okay. then probably, I will re-write it in a more natural way such as: "The general requirements for the implementation of  Status Attestation are listed in this section."